site/content/en/docs/formats/yolo8.md

```diff
@@ -0,0 +1,324 @@
+To create a Datumaro project with a YOLO8 source, use the following commands:
+
+```bash
+datum create
+datum import --format yolo8 <path/to/dataset> # for Detection dataset
+datum import --format yolo8_oriented_boxes <path/to/dataset> # for Oriented Bounding Box dataset
+datum import --format yolo8_segmentation <path/to/dataset> # for Segmentation dataset
+datum import --format yolo8_pose <path/to/dataset> # for Pose dataset
+```
+
+The YOLO8 dataset directory should have the following structure:
+
+```bash
+└─ yolo_dataset/
+   │ # a list of non-format labels (optional)  # file with list of classes
+   ├── data.yaml    # file with dataset information
+   ├── train.txt    # list of image paths in train subset [Optional]
+   ├── val.txt    # list of image paths in valid subset [Optional]
+   │
+   ├── images/
+   │   ├── train/  # directory with images for train subset
+   │   │    ├── image1.jpg
+   │   │    ├── image2.jpg
+   │   │    ├── image3.jpg
+   │   │    └── ...
+   │   ├── valid/  # directory with images for validation subset
+   │   │    ├── image11.jpg
+   │   │    ├── image12.jpg
+   │   │    ├── image13.jpg
+   │   │    └── ...
+   ├── labels/
+   │   ├── train/  # directory with annotations for train subset
+   │   │    ├── image1.txt
+   │   │    ├── image2.txt
+   │   │    ├── image3.txt
+   │   │    └── ...
+   │   ├── valid/  # directory with annotations for validation subset
+   │   │    ├── image11.txt
+   │   │    ├── image12.txt
+   │   │    ├── image13.txt
+   │   │    └── ...
+```
+
+`data.yaml` should have the following content:
+
+```yaml
+# Train/val/test sets as 1) dir: path/to/imgs, 2) file: path/to/imgs.txt, or 3) list: [path/to/imgs1, path/to/imgs2, ..]
+path:  ./ # dataset root dir
+train: train.txt  # train images (relative to 'path') 4 images
+val: val.txt  # val images (relative to 'path') 4 images
+
+# YOLO8 Pose specific field
+# First number is a number of points in skeleton
+# Second number defines a format of point info in an annotation txt files
+kpt_shape: [17, 3]
+
+# Classes
+names:
+  0: person
+  1: bicycle
+  2: car
+  # ...
+  77: teddy bear
+  78: hair drier
+  79: toothbrush
+```
+
+> Note, that though by default YOLO8 expects `data.yaml`,
+  Datumaro allows this file to have arbitrary name.
+
+Files `train.txt` and `val.txt` should have the following structure:
+
+```txt
+<path/to/image1.jpg>
+<path/to/image2.jpg>
+...
+```
+
+Files in directories `labels/train/` and `labels/valid/` should
+contain information about labels for images in `images/train` and `images/valid` respectively.
+If there are no objects in an image, no `.txt` file is required.
+
+Content of the `.txt` file depends on format.
+
+For **Detection** it contains bounding boxes:
+```txt
+# labels/image1.txt:
+# <label_index> <x_center> <y_center> <width> <height>
+0 0.250000 0.400000 0.300000 0.400000
+3 0.600000 0.400000 0.400000 0.266667
+...
+```
+
+For **Oriented Bounding Box** and **Segmentation** it contains coordinates
```

Comment:
I would separate OBB and Segmentation because a polygon can have more than 4 points. Does the original documentation also join them?

Reply:
separated them